app/src/main/java/org/schabi/newpipe/fragments/list/comments/CommentsFragment.java

```diff
@@ -74,7 +110,25 @@ protected Single<ListExtractor.InfoItemsPage<CommentsInfoItem>> loadMoreItemsLog
 
     @Override
     protected Single<CommentsInfo> loadResult(final boolean forceLoad) {
-        return ExtractorHelper.getCommentsInfo(serviceId, url, forceLoad);
+        if (replies == null) {
+            if (preComment == null) {
+                return ExtractorHelper.getCommentsInfo(serviceId, url, forceLoad);
+            } else {
+                return Single.fromCallable(() -> {
+                    // get a info template
+                    var info = ExtractorHelper.getCommentsInfo(
+                            serviceId, url, forceLoad).blockingGet();
+                    // clone comment object to avoid relatedItems and nextPage actually set null
+                    info = CommentUtils.clone(info);
```

Comment:
Mmmh, I don't like this that much. Can you explain further why you need to clone the object is a way that is so low-level?

Reply:
When you check `CommentReplyFragment`, you will know.
Because I need create a fake `CommentsInfo` for the `CommentFragment`, then I can simply put two `CommentFragment` together.